javascript-koans/koans/AboutApplyingWhatWeHaveLearnt.js

```diff
@@ -55,14 +55,31 @@ describe("About Applying What We Have Learnt", function() {
       }
     }
 
-    expect(sum).toBe(FILL_ME_IN);
+    expect(sum).toBe(233168);
   });
 
   it("should add all the natural numbers below 1000 that are multiples of 3 or 5 (functional)", function () {
+    var range = function (startNumber, endNumber) {
+      var resultArray = []
+      for(var i= 0; i<1000; i++){
```

Comment:
I think what you intended to write is this:
```
for( var i = startNumber; i < endNumber; i++ ){
// ...
```
I could be wrong. Speculating based on the function name.